snapshots/testsuite/testsuite.go

```diff
@@ -377,6 +377,7 @@ func snapshotterPrepareMount(ctx context.Context, snapshotter snapshots.Snapshot
 	if err = mount.All(mounts, preparing); err != nil {
 		return "", err
 	}
+	os.Chmod(preparing, 0755)
```

Comment:
This may not be needed. Do the tests fail in code spaces without it?

Reply:
Tests were failing without this change, however, updated dev config to make the tests pass without it https://github.com/containerd/containerd/pull/8295/commits/2182800e34e5aea380a8960dfb1d1db9529e3407